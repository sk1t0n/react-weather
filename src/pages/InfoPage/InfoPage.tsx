import React from 'react';
import Accordion from 'react-bootstrap/Accordion';

export const InfoPage: React.FC = () => {
  return (
    <div className="max-width-600_center">
      <Accordion defaultActiveKey="0">
        <Accordion.Item eventKey="0">
          <Accordion.Header>Weather source</Accordion.Header>
          <Accordion.Body>
            <h4>Free Weather API</h4>
            <a href="https://www.weatherapi.com/">https://www.weatherapi.com/</a>
          </Accordion.Body>
        </Accordion.Item>
      </Accordion>
    </div>
  );
};
