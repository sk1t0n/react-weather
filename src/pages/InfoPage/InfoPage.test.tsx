import { render } from '@testing-library/react';
import { InfoPage } from './InfoPage';

describe('InfoPage component', () => {
  it('check the snapshot', () => {
    const { asFragment } = render(<InfoPage />);
    expect(asFragment()).toMatchSnapshot();
  });
});
