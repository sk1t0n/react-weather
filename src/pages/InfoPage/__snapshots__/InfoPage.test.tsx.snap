// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InfoPage component check the snapshot 1`] = `
<DocumentFragment>
  <div
    class="max-width-600_center"
  >
    <div
      class="accordion"
    >
      <div
        class="accordion-item"
      >
        <h2
          class="accordion-header"
        >
          <button
            aria-expanded="true"
            class="accordion-button"
            type="button"
          >
            Weather source
          </button>
        </h2>
        <div
          class="accordion-collapse collapse show"
        >
          <div
            class="accordion-body"
          >
            <h4>
              Free Weather API
            </h4>
            <a
              href="https://www.weatherapi.com/"
            >
              https://www.weatherapi.com/
            </a>
          </div>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
